js/core/index.js

```diff
@@ -20,10 +20,12 @@ require('./polyfill');
 const random = require('./random');
 const block = require('./block');
 const keyboard = require('./keyboard');
-const ps2 = require('./ps2');
-const pci = require('./pci');
+const ps2 = require('./arch/'+process.arch+'/ps2');
+const pci = require('./arch/'+process.arch+'/pci');
 const net = require('./net');
 const stdio = require('./stdio');
+const tty = require('./tty');
+const shell = require('./shell');
```

Comment:
All of these are drivers, so same as before. In fact, the `shell` is not a driver itself, but instead it's an app, so don't merge them.